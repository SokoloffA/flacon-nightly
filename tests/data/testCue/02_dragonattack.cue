FILE "dragonattack.wav" WAVE
PERFORMER "Various"
TITLE "Dragon Attack - A Tribute to Queen"
  TRACK 01 AUDIO
    TITLE "McAuley, Impellitteri, Kulick, Roth, Phillips, Schellen / I Want It All"
    INDEX 00 00:00:00
    INDEX 01 00:00:30
  TRACK 02 AUDIO
    TITLE "LaBrie, Friedman, Franklin, Appice / Sheer Heart Attack"
    INDEX 01 04:08:02
  TRACK 03 AUDIO
    TITLE "Adam, Petrucci, Sarzo, Aldridge / Another One Bites the Dust"
    INDEX 01 07:35:59
  TRACK 04 AUDIO
    TITLE "Sato, Kulick, Kulick, Sherwood, Phillips, Singer / Save Me"
    INDEX 01 11:01:62
  TRACK 05 AUDIO
    TITLE "Shortino, Kulick, Schellen, Laurent, Dio / We Will Rock You"
    INDEX 01 15:13:54
  TRACK 06 AUDIO
    TITLE "Shortino, Krieger, Kulick, Sherwood, Franklin, Appice, Laurent, Dio / We are the Champions"
    INDEX 01 17:16:21
  TRACK 07 AUDIO
    TITLE "Kilmister, Nugent, Kilmister, Kulick, Sarzo, Aldridge / Tie Your Mother Down"
    INDEX 01 20:57:08
  TRACK 08 AUDIO
    TITLE "Hughes, Lee, Kulick, Franklin, Appice / Get Down Make Love"
    INDEX 01 24:43:15
  TRACK 09 AUDIO
    TITLE "Boals, Malmsteen, Sarza, Aldridge / Keep Yourself Alive"
    INDEX 01 28:53:69
  TRACK 10 AUDIO
    TITLE "LaBrie, Bouillet, Bouillet, Kulick, Phillips, Schellen / One Vision"
    INDEX 01 32:42:60
  TRACK 11 AUDIO
    TITLE "Bush, Ian, Throne, Vera, Ian / It's Late"
    INDEX 01 36:27:64
