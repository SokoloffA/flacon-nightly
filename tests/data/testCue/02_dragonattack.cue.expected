[ DISC 01 / TRACK 01 ]
	FILE        = dragonattack.wav
	INDEX CD 00 = 00:00:00
	INDEX CD 01 = 00:00:30
	TITLE       = I Want It All
	ALBUM       = Dragon Attack - A Tribute to Queen
	PERFORMER   = McAuley, Impellitteri, Kulick, Roth, Phillips, Schellen
	DATE        =
	DISCID      =
	TRACKNUM    = 1
	TRACKCOUNT  = 11
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 02 ]
	FILE        = dragonattack.wav
	INDEX CD 00 = 04:08:02
	INDEX CD 01 = 04:08:02
	TITLE       = Sheer Heart Attack
	ALBUM       = Dragon Attack - A Tribute to Queen
	PERFORMER   = LaBrie, Friedman, Franklin, Appice
	DATE        =
	DISCID      =
	TRACKNUM    = 2
	TRACKCOUNT  = 11
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 03 ]
	FILE        = dragonattack.wav
	INDEX CD 00 = 07:35:59
	INDEX CD 01 = 07:35:59
	TITLE       = Another One Bites the Dust
	ALBUM       = Dragon Attack - A Tribute to Queen
	PERFORMER   = Adam, Petrucci, Sarzo, Aldridge
	DATE        =
	DISCID      =
	TRACKNUM    = 3
	TRACKCOUNT  = 11
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 04 ]
	FILE        = dragonattack.wav
	INDEX CD 00 = 11:01:62
	INDEX CD 01 = 11:01:62
	TITLE       = Save Me
	ALBUM       = Dragon Attack - A Tribute to Queen
	PERFORMER   = Sato, Kulick, Kulick, Sherwood, Phillips, Singer
	DATE        =
	DISCID      =
	TRACKNUM    = 4
	TRACKCOUNT  = 11
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 05 ]
	FILE        = dragonattack.wav
	INDEX CD 00 = 15:13:54
	INDEX CD 01 = 15:13:54
	TITLE       = We Will Rock You
	ALBUM       = Dragon Attack - A Tribute to Queen
	PERFORMER   = Shortino, Kulick, Schellen, Laurent, Dio
	DATE        =
	DISCID      =
	TRACKNUM    = 5
	TRACKCOUNT  = 11
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 06 ]
	FILE        = dragonattack.wav
	INDEX CD 00 = 17:16:21
	INDEX CD 01 = 17:16:21
	TITLE       = We are the Champions
	ALBUM       = Dragon Attack - A Tribute to Queen
	PERFORMER   = Shortino, Krieger, Kulick, Sherwood, Franklin, Appice, Laurent, Dio
	DATE        =
	DISCID      =
	TRACKNUM    = 6
	TRACKCOUNT  = 11
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 07 ]
	FILE        = dragonattack.wav
	INDEX CD 00 = 20:57:08
	INDEX CD 01 = 20:57:08
	TITLE       = Tie Your Mother Down
	ALBUM       = Dragon Attack - A Tribute to Queen
	PERFORMER   = Kilmister, Nugent, Kilmister, Kulick, Sarzo, Aldridge
	DATE        =
	DISCID      =
	TRACKNUM    = 7
	TRACKCOUNT  = 11
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 08 ]
	FILE        = dragonattack.wav
	INDEX CD 00 = 24:43:15
	INDEX CD 01 = 24:43:15
	TITLE       = Get Down Make Love
	ALBUM       = Dragon Attack - A Tribute to Queen
	PERFORMER   = Hughes, Lee, Kulick, Franklin, Appice
	DATE        =
	DISCID      =
	TRACKNUM    = 8
	TRACKCOUNT  = 11
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 09 ]
	FILE        = dragonattack.wav
	INDEX CD 00 = 28:53:69
	INDEX CD 01 = 28:53:69
	TITLE       = Keep Yourself Alive
	ALBUM       = Dragon Attack - A Tribute to Queen
	PERFORMER   = Boals, Malmsteen, Sarza, Aldridge
	DATE        =
	DISCID      =
	TRACKNUM    = 9
	TRACKCOUNT  = 11
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 10 ]
	FILE        = dragonattack.wav
	INDEX CD 00 = 32:42:60
	INDEX CD 01 = 32:42:60
	TITLE       = One Vision
	ALBUM       = Dragon Attack - A Tribute to Queen
	PERFORMER   = LaBrie, Bouillet, Bouillet, Kulick, Phillips, Schellen
	DATE        =
	DISCID      =
	TRACKNUM    = 10
	TRACKCOUNT  = 11
	DISCNUM     = 1
	DISCCOUNT   = 1


[ DISC 01 / TRACK 11 ]
	FILE        = dragonattack.wav
	INDEX CD 00 = 36:27:64
	INDEX CD 01 = 36:27:64
	TITLE       = It's Late
	ALBUM       = Dragon Attack - A Tribute to Queen
	PERFORMER   = Bush, Ian, Throne, Vera, Ian
	DATE        =
	DISCID      =
	TRACKNUM    = 11
	TRACKCOUNT  = 11
	DISCNUM     = 1
	DISCCOUNT   = 1
